jquery.dynatable.js

```diff
@@ -967,6 +967,42 @@
         // force false boolean value to -1, true to 1, and tie to 0
         return comparison === false ? -1 : (comparison - 0);
       },
+      alphaNumeric: function(a, b, attr, direction) {
+        //swap if reverse
+        if(direction === -1) {
+          var temp = a;
+          a = b;
+          b = temp;
+        }
+
+        //extract display text
+        a = $(a[attr.toString()]).text();
```

Comment:
Drop the `toString()` on the attr names, since names are always strings anyway regardless of using integers, etc to refer to them.